pkgs/os-specific/linux/libbpf/default.nix

```diff
@@ -33,7 +33,7 @@ stdenv.mkDerivation rec {
     description = "Upstream mirror of libbpf";
     homepage    = "https://github.com/libbpf/libbpf";
     license     = with licenses; [ lgpl21 /* or */ bsd2 ];
-    maintainers = with maintainers; [ thoughtpolice vcunat saschagrunert ];
+    maintainers = with maintainers; [ thoughtpolice vcunat ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ thoughtpolice vcunat saschagrunert ];
```

Reply:
Good point, I just blindly reverted the commit but will amend to keep @saschagrunert there.